example/spec/parallel/testAPI/bulkOperationsSuccessSpec.js

```diff
@@ -273,10 +275,19 @@ describe('POST /granules/bulk', () => {
         } catch (error) {
           throw new SyntaxError(`getAsyncOperationBody.output is not valid JSON: ${getAsyncOperationBody.output}`);
         }
+
+        await getGranuleWithStatus({
```

Comment:
This PR updates this test to await the granule record existing with a later record than the original ingest and a status of `running`. 

*That* is because the workflow this test is triggering doesn't report a granule as finished.    Open question as to if we want to add another workflow (I may add it to the PR and OBE this comment shortly) but it's not needed to meet the objective of removing the bad write garbage from the log.

Reply:
Apparently this isn't the only trigger here.  🤔